packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -14,18 +14,21 @@ const propTypes = {
   /**
   * Object of configuration for the side navigation and profile.
   */
+  // eslint-disable-next-line react/forbid-prop-types
   nav: PropTypes.object.isRequired,
```

Comment:
Opted to simply disable lint on a line-by-line basis instead of importing in the prop shapes (same in other cases below)

Reply:
any reason for not doing `// eslint-disable-line` ?

Reply:
As in
`navItems: PropTypes.array, // eslint-disable-line react/forbid-prop-types`
or just
`navItems: PropTypes.array, // eslint-disable-line`
?